### ISSUE TITLE
L'injection de dépendance avec Effect: comment créer et utiliser des services

### ISSUE THREAD

**Issue by vinassefranche** [evryg-org/effect-paris#3]
### Format

Long (20 minutes)

### Description

- Comment créer un service injectable dans le monde d'Effect
- Comment utiliser ce service dans une fonction 
- Comment injecter une implementation du service pour exécuter cette fonction 
- Comment l'injection de dépendance nous permet de tester cette fonction facilement

### Email

vincent.francois@inato.com

**Comment by antoine-coulon** [comment]
Hello @vinassefranche, super le talk !

Je te proposerai une date dès que j'ai plus de visibilité sur la planification de la prochaine édition.

Cheers

**Comment by vinassefranche** [comment]
Hello @antoine-coulon 
Je ne pourrais malheureusment pas pouvoir faire mon talk la semaine du 4 novembre car on a déjà un évènement de prévu cette semaine à Inato.
J'espère que ça sera ok pour la prochaine session!

**Comment by antoine-coulon** [comment]
Hello @vinassefranche,

Aucun problème ça pourra être planifié pour la prochaine session. Je reviendrai vers toi ici quand on aura de la visibilité sur la date de l'édition d'après

**Comment by antoine-coulon** [comment]
Hello @vinassefranche, 

Est-ce que la période entre mi-janvier et fin janvier serait jouable pour toi ?

**Comment by vinassefranche** [comment]
@antoine-coulon a priori pas de soucis. Ca dépendra forcément du jour précis mais il n'y a rien dans cette période qui rentre en conflit donc ça devrait le faire!

**Comment by antoine-coulon** [comment]
@vinassefranche super et je comprends, je te tiendrai bien évidemment informé une fois que la date précise sera connue, mais a priori ce sera sur cette période.

Bonne soirée et à bientôt !

**Comment by antoine-coulon** [comment]
Hello @vinassefranche, ton talk est prévu pour le 21 janvier ! Bonnes fêtes de fin d'année et à bientôt :)

**Comment by vinassefranche** [comment]
Yes. Jérémie m'avait prévenu aussi! Merci et bonnes fêtes à toi aussi!
